proposals/p1983.md

```diff
@@ -0,0 +1,183 @@
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+[Pull request](https://github.com/carbon-language/carbon-lang/pull/1983)
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Abstract](#abstract)
+-   [Problem](#problem)
+-   [Background](#background)
+-   [Proposal](#proposal)
+-   [Rationale](#rationale)
+-   [Alternatives considered](#alternatives-considered)
+    -   [3-digit decimal groupings](#3-digit-decimal-groupings)
+    -   [2-digit or 4-digit hexadecimal digit groupings](#2-digit-or-4-digit-hexadecimal-digit-groupings)
+    -   [Disallow digit separators in fractions](#disallow-digit-separators-in-fractions)
+
+<!-- tocstop -->
+
+## Abstract
+
+[Proposal #143: Numeric literals](/proposals/p0143.md) added digit separators
+with strict rules for placement. It missed some use-cases. In order to address
+this, remove placement rules for numeric literals.
+
+## Problem
+
+Digit separator placement rules may be too strict.
+
+## Background
+
+[Proposal #143: Numeric literals](/proposals/p0143.md) added digit separators
+with strict rules for placement:
+
+-   For decimal integers, the digit separators shall occur every three digits
+    starting from the right. For example, `2_147_483_648`.
+-   For hexadecimal integers, the digit separators shall occur every four digits
+    starting from the right. For example, `0x7FFF_FFFF`.
+-   For real number literals, digit separators can appear in the decimal and
+    hexadecimal integer portions (prior to the period and after the optional `e`
+    or mandatory `p`) as described in the previous bullets. For example,
+    `2_147.483648e12_345` or `0x1_00CA.FEF00Dp+24`
+-   For binary literals, digit separators can appear between any two digits. For
+    example, `0b1_000_101_11`.
+
+Some issues with this approach have been raised:
+
+-   For integers, while the original proposal mentioned Indian digit groupings,
+    Chinese, Japanese, and Korean cultures use 4-digit groupings. This oversight
+    is likely due to the description on
+    [wikipedia](https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping):
+    "but the delimiter commonly separates every three digits".
+-   For hexadecimal in particular, there are microformats where different
+    placement rules may be desirable. For example, MAC addresses are typically
+    in byte pairs (`00-B0-D0-63-C2-26`) and UUIDs have particular groupings
+    (`123e4567-e89b-12d3-a456-426614174000`).
+
+This was asked on
+[Issue #1485: Reconsider digit separators](https://github.com/carbon-language/carbon-lang/issues/1485),
+where a proposal was requested.
+
+## Proposal
+
+Switch to simple rules: a single digit separator can appear between any two
+digits. For example:
+
+-   Decimal literals: `1_2_3`
+-   Hexadecimal literals: `0xA_B_C`
+-   Real literals: `1_2.3_4e5_6`
+-   Binary literals are unaffected.
+
+## Rationale
```

Comment:
Could you please include the specifics of the rationale from #1485 here, like China, MAC addresses, UUID, ISBN, credit card numbers, etc.?